app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -207,7 +207,7 @@ def consumer_create
       content_view_environment = find_content_view_environment
       host = Katello::Host::SubscriptionFacet.find_or_create_host(content_view_environment.environment.organization, rhsm_params)
 
-      sync_task(::Actions::Katello::Host::Register, host, rhsm_params, content_view_environment)
+      Katello::RegistrationManager.register_host(host, rhsm_params, content_view_environment)
```

Comment:
we need to make sure we have a ping check on both of these to ensure that all the services are functional

Reply:
and we could add it in the controller

Reply:
I think I'll add it to `register_host` and `unregister_host` just because there are a couple of entry points in the contorller (AKey, username/pass registration, etc)

Reply:
I ended up adding to the controller after all 😸